alacritty_terminal/src/tty/unix.rs

```diff
@@ -173,12 +175,20 @@ fn default_shell_command(shell: &str, user: &str) -> Command {
     // `login` normally does this itself, but `-l` disables this.
     let exec = format!("exec -a -{} {}", shell_name, shell);
 
+    // Since we use -l, `login` will not change directory to the user's home. However,
+    // `login` only checks the current working directory for a .hushlogin file, causing
+    // it to miss any in the user's home directory. We can fix this by doing the check
+    // ourselves and passing `-q`
+    let has_home_hushlogin = Path::new(home).join(".hushlogin").exists();
```

Comment:
One question that comes to my mind reading this is if this is generally only a home dir thing, or if a global option exists as well? Is there an option to put this file somewhere else to disable it for all users? Since this seems pretty hacky, I'd assume not?

Reply:
The man pages for `login` don't mention one: https://ss64.com/mac/login.html